isomorphe/batch.py

```diff
@@ -1,9 +1,58 @@
 from dataclasses import dataclass
 from enum import IntEnum, StrEnum
+from typing import Iterator
+
+from lxml.etree import _ListErrorLog as ETListErrorLog
 
 from isomorphe.geonetwork import MefArchive, MetadataType, WorkflowState
 
 
+@dataclass(kw_only=True)
+class TransformError:
+    message: str
+    line: int
+    column: int
+    domain_name: str
+    domain: int
+    type_name: str
+    type: int
+    level_name: str
+    level: int
+    filename: str
+
+
+class TransformErrorLog:
```

Comment:
Nitpick, do we keep "error log" knowing it's not just errors?

Reply:
Yeah, same. Do you have a suggestion? `TransformWarnings`? `TransformMessages`?

Reply:
Warning goes with Avertissement. And is compatible with our log level filter.

Reply:
The log level is "warnings or worse". I'm not sure if errors might creep in or not. I'm thinking no since we abort on errors, but who knows?
`Messages` is the safer bet IMO, but no strong opinion.

Reply:
I went with a neutral `Log` / `LogItem`